ios/magnet/Base.lproj/LaunchScreen.xib

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I have no idea what these changes do, but they were introduced by the RN upgrade.